src/Pages/ProductDetail/ProductDetail.js

```diff
@@ -41,6 +42,9 @@ const ProductDetail = () => {
           </TasteSummary>
         </FlexRow>
       </TasteWrapper>
+      {product.image !== '' && (
```

Comment:
```suggestion
      {product.image && (
```
굳이 빈문자열이랑 비교할 필요가 없어보입니다!

Reply:
넵 수정하였습니다 !